dev/com.ibm.ws.webcontainer.servlet.4.0/src/com/ibm/ws/webcontainer40/osgi/webapp/WebApp40.java

```diff
@@ -76,7 +73,7 @@ public WebApp40(WebAppConfiguration webAppConfig,
     @Override
     public int getMajorVersion() {
         int level = WebContainer.getServletContainerSpecLevel();
-        return level == WebContainer.SPEC_LEVEL_60 ? 6 : level == WebContainer.SPEC_LEVEL_50 ? 5 : 4;
+        return level == WebContainer.SPEC_LEVEL_61 ? 61 : level == WebContainer.SPEC_LEVEL_60 ? 6 : level == WebContainer.SPEC_LEVEL_50 ? 5 : 4;
```

Comment:
I think this is a type and SPEC_LEVEL_60 || SPEC_LEVEL_61 the major version should be 6.

Reply:
We also will need an update for the minor version.

Reply:
WebContainer.getServletContainerSpecLevel() will return an int from 

_dev/io.openliberty.webcontainer.servlet.6.1.internal.factories/src/com/ibm/ws/webcontainer/speclevel/servletSpecLevel.properties_

Reply:
Updated WebApp40.  If we have a new WebApp60 later on, I will move these 2 getter version to the new class.